### ISSUE TITLE
I THINK I fixed the nodes losing connection issue, also another issue with mxSeed

### ISSUE THREAD

**Issue by lordwedggie** [Smirnov75/ComfyUI-mxToolkit#25]
I think the connection issue is caused by your dynamic switch between float and int? I couldn't find any actual errors so I rewrote your code and divided the sliders to INT and FLOAT, also added a fill bar and a bunch of customization options, it's at: [https://github.com/lordwedggie/xcpNodes]

Your mxSeed is awesome! I'm using it in all my workflows, however, upon loading a new workflow, the first time you click on the seed it doesn't create a new seed, but launches the queue on the same seed again? It's a minor issue but hopefully it can be fixed. Cheers!

**Comment by Smirnov75** [comment]
Yes, it's all due to the int-float switch. There have been issues with this before. Fortunately, the new version of ComfyUI offers more options to help prevent such errors. So, for now, I don't plan to split the nodes, even though it's the easiest and most reliable solution.
I've also noticed the mxSeed bug. I'll try to fix it in the near future.
P.S. I'm really glad that my nodes are useful to you.

**Comment by lordwedggie** [comment]
Glad to hear! I'm making a bunch of minimalist sliders and bool buttons, aim is to take as little screen space as possible, all based on your nodes, do you mind if I put them up to share? You will of course be credited and your GitHub linked. Oh btw I've been trying to change the node back ground color, but can only make it work in this.node.onAdded, is there a work around? Thanks and...CHEERS!

**Comment by Smirnov75** [comment]
My nodes are distributed under the GPLv3 license, so there are no issues with freely using my code in other projects.
Try calling app.graph.setDirtyCanvas(true, true); after changing the bgcolor. This might work in some cases.

**Comment by lordwedggie** [comment]
Hi, sorry to bother you again! I was trying to migrate my sliders code to your router code, by using `LiteGraph.registerNodeType ` instead of `async beforeRegisterNodeDef`, since with it I could use` mynode.title_mode = 1` to remove the title. However, for the life of me I couldn't figure out how to have the node have an output value? I'm assuming it's because it doesn't have the python class, but if I add the corresponding python code, the JS no longer works.
Adding title_mode to my normal nodes has no effect (I tried adding 'extend LGraphNode', same thing),` LGraphNode.title_mode = 1` works, but it turns the title off on every node....Do you have any idea how to fix this? I'm pulling my hair out here ;0...
